src/app/components/util/chpl-ellipsis.jsx

```diff
@@ -27,25 +44,25 @@ function ChplEllipsis(props) {
   }
 
   return (
-    <span>
+    <>
       {isShortened ? display : text}
       {display !== text && isShortened
        && (
          <ChplTooltip title={text}>
-           <button type="button" className="btn btn-link btn-xs" onClick={() => setShortened(false)}>
-             <i className="fa fa-ellipsis-h" />
+           <IconButton size="small" className={classes.chplEllipsis}  onClick={() => setShortened(false)}>
```

Comment:
You don't have to, but once a React Element has three attributes on it, I usually linebreak them. It's helpful when looking at changes if a single line changes, but in that line there's only one thing

```suggestion
           <IconButton 
               size="small" 
               className={classes.chplEllipsis}  
               onClick={() => setShortened(false)}
           >
```

With a change like the above, if we later want to change the size to "normal" or whatever, it would be just one line that changed, but it would be clear we were only changing the size, and not the `onClick` functionality or something